interfaces/screen-capture.idl

```diff
@@ -0,0 +1,27 @@
+// GENERATED CONTENT - DO NOT EDIT
+// Content of this file was automatically extracted from the
+// "Screen Capture" spec.
+// See: https://w3c.github.io/mediacapture-screen-share/
+
+partial interface Navigator {
+    Promise<MediaStream> getDisplayMedia(optional MediaStreamConstraints constraints);
+};
+
+partial dictionary MediaTrackConstraintSet {
+             ConstrainDOMString displaySurface;
+             ConstrainBoolean logicalSurface;
+             ConstrainDOMString cursor;
```

Comment:
So no reformatting on ingest?

Reply:
No, lots of these IDL files look a bit funny as a result. This would be a feature request on https://github.com/tidoust/reffy/issues, IDL is imported verbatim from https://github.com/tidoust/reffy-reports.